plugins/ipam/dhcp/daemon.go

```diff
@@ -37,6 +37,7 @@ const listenFdsStart = 3
 const resendCount = 3
 
 var errNoMoreTries = errors.New("no more tries")
+var hostNetnsPrefix = ""
```

Comment:
This should be a member of the `DHCP` struct, not a package variable

Reply:
Currently one DHCP daemon serves one DHCP structure hence I don't think so, but I am okey to do that because in the future DHCP daemon may serves multiple DHCP structure.

I will update the diff.